tbd.schema.json

```diff
@@ -315,6 +322,27 @@
         "surfaces"
       ]
     },
+    "SubSurface": {
+      "description": "Suburface(s) U-factors (optional)",
+      "type": "object",
+      "properties": {
+        "id": {
+          "title": "e.g. OS or E+ surface identifier",
+          "type": "string"
+        },
+        "usi": {
+          "title": "Overall U-factor (optional)",
+          "type": "number",
+          "minimum": 0.01,
+          "maximum": 10.0
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "id",
+        "usi"
```

Comment:
Is `usi` optional or required?  Title says optional.

Reply:
Thanks for pointing that out. It's required - it's the only reason TBD has a subsurface entry. Will fix.